app-emacs/ace-window/ace-window-0.10.0_p20200706.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+NEED_EMACS="24.1"
+
+inherit elisp
+
+DESCRIPTION="GNU Emacs package for selecting a window to switch to."
```

Comment:
Random thought: if we're using a snapshot, maybe we should put a comment in the ebuild saying why, so people don't get confused.